### DISCUSSION TITLE
How to make sure that a host can run lotus daemon

### DISCUSSION THREAD

**Discussion by de-authority** [filecoin-project/lotus#6719] [Storage Provider] [answered]
Chain synchronization is no longer a problem for many miners, but it is still difficult for many new miners and developers. What factors do we need to check to ensure that their basic environment meets the requirements?

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
the official requirements can be found here: https://docs.filecoin.io/get-started/lotus/installation/#minimal-requirements
